README.md

```
@@ -2,313 +2,1521 @@
+- [Vishnu Sharma](https://vs-code-cloud.github.io)
+- [Vishwanath](https://frozenhearth.vercel.app)
+- [Vishwasa Navada](https://vishwas.tech)
+- [Vitaliy Ivanov](https://vitaliy.vercel.app)
+- [Vito Sartori](https://vito.io)
+- [Vitor Forbrig](https://forbrig.github.io)
+- [Vivek Choudhary](https://vivekportfoliopersonal.vercel.app) [Full Stack Engineer | AI Engineer]
+- [Vivek Chudasama](https://vivekchudasama-2004.github.io/portfolio)
+- [Vivek Jaiswal](https://www.vivekjaiswal.me) [Software Archutect | Ai/Ml Expert]
+- [Vivek Patel Ubuntu](http://vivek9patel.github.io)
+- [Vivek Patel](http://vivek9patel.com)
+- [Vivek Raval](https://mrthree.netlify.app)[App | Ios | Web | Ai & Ml Engineer]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://mrthree.netlify.app)